api/src/main/java/org/pmiops/workbench/api/OfflineObjectNameSizeController.java

```diff
@@ -0,0 +1,31 @@
+package org.pmiops.workbench.api;
+
+import java.util.logging.Logger;
+import org.pmiops.workbench.exfiltration.ObjectNameLengthService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+/** Controller to audit users who created objects in their buckets with very long names */
+public class OfflineObjectNameSizeController implements OfflineObjectNameSizeApiDelegate {
+
+  private static final Logger LOGGER =
```

Comment:
I don't know why but everyelse uses `logger`

Reply:
I prefer to use LOGGER in my code, because it's static final, is there a specific naming convention for loggers for our codebase. I think the logger naming is always a controversial topic :)